### PR TITLE
FIX-#3010: Update Ray version to 1.3

### LINKED ISSUE #3010: Bump Ray Version to 1.3

### PR DESCRIPTION
**Pull request by devin-petersohn**
Signed-off-by: Devin Petersohn <devin.petersohn@gmail.com>

## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #3010 
- [ ] tests added and passing

### CODE REVIEW

**Comment by YarShev** [comment]
Restarted `Containerized Modin on Ray tests`.

**Comment by YarShev** [comment]
This PR duplicates #2762 except some changes in the code and in the environment files. @devin-petersohn , do you plan to make respective changes in this PR?

**Comment by devin-petersohn** [comment]
@YarShev this is not a duplicate of #2762. Ray 1.2 did not work and now that Ray 1.3 is out, we should update. 

We still have an issue with CI not checking PyPI wheels and the conda forge build will always lag. We should check with PyPI unless we plan to constantly keep https://github.com/conda-forge/ray-packages-feedstock up to date ourselves. So far, it is behind by 2 releases now.

**Comment by YarShev** [comment]
I see, that makes sense.

File: modin/config/envvars.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def _get_default(cls):
         except ImportError:
             pass
         else:
-            if version.parse(ray.__version__) < version.parse("1.0.0"):
+            if version.parse(ray.__version__) < version.parse("1.3.0"):
```
Comment (YarShev) [inline modin/config/envvars.py:68]:
```suggestion
            if version.parse(ray.__version__) <= version.parse("1.2.0"):
```

File: modin/engines/ray/utils.py [hunk 2]
```diff
@@ -167,13 +167,7 @@ def initialize_ray(
                 "_redis_password": redis_password,
                 "logging_level": 100,
                 "_memory": object_store_memory,
-                "_lru_evict": True,
```
Comment (YarShev) [inline modin/engines/ray/utils.py:170]: Why did you remove this?

File: modin/config/envvars.py (hunk 1 above)
Comment (YarShev) [reply modin/config/envvars.py:68]: We can't require ray version < 1.3.0 because we use environment-dev.yml, which installs 1.0.0, in CI.

**Comment by vnlitvinov** [comment]
Shouldn't we wait until https://github.com/conda-forge/ray-packages-feedstock/pull/11 (and hence Ray 1.3.0 available in `conda-forge`) before continuing?

File: modin/engines/ray/utils.py (hunk 2 above)
Comment (devin-petersohn) [reply modin/engines/ray/utils.py:170]: That parameter is no longer supported in Ray init. That is why we had special casing for the code immediately below

**Comment by devin-petersohn** [comment]
We should just use pypi because we shouldn't delay the upgrade. There are a lot of features on the latest Ray that we will want to take advantage of.

**Comment by vnlitvinov** [comment]
> We should just use pypi because we shouldn't delay the upgrade. There are a lot of features on the latest Ray that we will want to take advantage of.

cc @aregm 

This effectively means we're breaking the ability for end users to one-command-install latest Modin with OmniSci... as OmniSci is not present in pypi AFAIK.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -253,23 +253,23 @@ jobs:
...
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_reduction.py --backend=${{ matrix.backend }}
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_udf.py --backend=${{ matrix.backend }}
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_window.py --backend=${{ matrix.backend }}
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_pickle.py --backend=${{ matrix.backend }}
-      - run: python -m pytest -n 2 modin/pandas/test/test_series.py --backend=${{ matrix.backend }}
-      - run: python -m pytest -n 2 modin/pandas/test/test_rolling.py --backend=${{ matrix.backend }}
-      - run: python -m pytest -n 2 modin/pandas/test/test_concat.py --backend=${{ matrix.backend }}
-      - run: python -m pytest -n 2 modin/pandas/test/test_groupby.py --backend=${{ matrix.backend }}
-      - run: python -m pytest -n 2 modin/pandas/test/test_reshape.py --backend=${{ matrix.backend }}
-      - run: python -m pytest -n 2 modin/pandas/test/test_general.py --backend=${{ matrix.backend }}
```
Comment (gshimansky) [inline .github/workflows/ci.yml:272]: Could you please make changes to tests in a separate PR as we usually required? Also, what is the reason to disable parallel tests?

Comment (devin-petersohn) [reply .github/workflows/ci.yml:272]: I saw some segmentation faults happened because of parallel test issues with the latest Ray. If it is necessary to keep the changes I will make a new PR, but for now it was meant to just test.

Comment (gshimansky) [reply .github/workflows/ci.yml:272]: I really hope that we can keep parallel tests. If new version of Ray cannot run in parallel, it would mean a regression from the current state.

Comment (devin-petersohn) [reply .github/workflows/ci.yml:272]: @gshimansky I agree

**Comment by gshimansky** [comment]
Ray bug that reproduces CI crashes https://github.com/ray-project/ray/issues/15990

**Comment by devin-petersohn** [comment]
@gshimansky In this case, I suggest that we skip the failing tests in Ray until that bug can be fixed. As long as TeamCity can pass it will be fine. TeamCity is not currently passing, which is perhaps due to the xdist issue.

**Comment by gshimansky** [comment]
> @gshimansky In this case, I suggest that we skip the failing tests in Ray until that bug can be fixed. As long as TeamCity can pass it will be fine. TeamCity is not currently passing, which is perhaps due to the xdist issue.

If xdist cannot be enabled, we better just disable TeamCity CI because single process it will take forever to complete. TeamCity CI runs on big fat 48 core servers, but there are only a few of them. Parallelizing tasks by hosts is not an option, only parallelizing by processes running on the same host.

**Comment by devin-petersohn** [comment]
@gshimansky based on the TeamCity logs do you think it is an xdist problem?

**Comment by gshimansky** [comment]
> @gshimansky based on the TeamCity logs do you think it is an xdist problem?

It looks to me like the same crash bug in ray https://github.com/ray-project/ray/issues/15990 that was just fixed in Ray master. The reason for it is that a list of crashed tests looks very similar to what we have in Github Actions CI:
<details>
<pre>
[09:20:25]	[Step 5/8] =================================== FAILURES ===================================
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw46] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw46' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[rsub-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw47] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw47' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[sub-3.0-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw38] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw38' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[sub-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw41] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw41' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rsub-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw22] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw22' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rtruediv-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw43] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw43' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[rmod-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw33] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw33' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[sub-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw3] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw3' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rmod-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw18] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw18' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[truediv-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw10] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw10' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[truediv-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw15] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw15' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[rtruediv-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw13] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw13' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[mul-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw40] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw40' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[add-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw26] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw26' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[radd-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw28] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw28' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rmul-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw45] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw45' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[radd-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw2] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw2' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[mod-3.0-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw4] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw4' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rmod-3.0-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw31] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw31' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[add-columns-series_or_list]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw29] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw29' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[mul-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw0] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw0' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[mod-None-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw20] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw20' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[rmul-columns-series_or_list]'
[09:20:25]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:25]	[Step 5/8] [gw5] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw5' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-1-True-<lambda>0]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw55] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw55' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rsub-3.0-check_different_index]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw49] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw49' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[add-3.0-check_different_index]'
[09:20:25]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:25]	[Step 5/8] [gw68] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw68' crashed while running 'modin/pandas/test/test_groupby.py::test_agg_func_None_rename[False-by_and_agg_dict2]'
[09:20:25]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:25]	[Step 5/8] [gw17] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw17' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-1-False-<lambda>6]'
[09:20:25]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:25]	[Step 5/8] [gw59] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:25]	[Step 5/8] worker 'gw59' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[radd-3.0-check_different_index]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw16] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw16' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[truediv-3.0-check_different_index]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw24] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw24' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[rfloordiv-columns-series_or_list]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw12] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw12' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rfloordiv-None-check_different_index]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw56] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw56' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions[floordiv-columns-series_or_list]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw63] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw63' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rtruediv-3.0-check_different_index]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw39] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw39' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-1-True-<lambda>7]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw34] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw34' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-1-False-<lambda>0]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw44] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw44' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[floordiv-3.0-check_different_index]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw37] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw37' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-1-True-<lambda>6]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw64] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw64' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-1-False-<lambda>7]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw42] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw42' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-2-True-<lambda>6]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw54] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw54' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-2-False-<lambda>6]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw32] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw32' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-2-True-<lambda>0]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw61] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw61' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rfloordiv-3.0-check_different_index]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw19] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw19' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-2-False-<lambda>0]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw60] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw60' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[rmul-3.0-check_different_index]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw48] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw48' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[mul-3.0-check_different_index]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw50] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw50' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-2-False-<lambda>7]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw67] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw67' crashed while running 'modin/pandas/test/test_groupby.py::test_multi_column_groupby_different_partitions[False-2-True-<lambda>7]'
[09:20:26]	[Step 5/8] __________________ modin/pandas/test/dataframe/test_binary.py __________________
[09:20:26]	[Step 5/8] [gw73] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw73' crashed while running 'modin/pandas/test/dataframe/test_binary.py::test_math_functions_fill_value[floordiv-None-check_different_index]'
[09:20:26]	[Step 5/8] ______________________ modin/pandas/test/test_groupby.py _______________________
[09:20:26]	[Step 5/8] [gw9] linux -- Python 3.8.10 /root/anaconda3/envs/modin/bin/python
[09:20:26]	[Step 5/8] worker 'gw9' crashed while running 'modin/pandas/test/test_groupby.py::test_agg_func_None_rename[True-by_and_agg_dict2]'
</pre>
</details>

**Comment by h-vetinari** [comment]
Hey, just saw this linked from https://github.com/conda-forge/ray-packages-feedstock/pull/11 which was auto-closed after @vnlitvinov managed to merge https://github.com/conda-forge/ray-packages-feedstock/pull/8. Do you still need 1.3 specifically, or is going to 1.4 also an option?

**Comment by vnlitvinov** [comment]
> Hey, just saw this linked from [conda-forge/ray-packages-feedstock#11](https://github.com/conda-forge/ray-packages-feedstock/pull/11) which was auto-closed after @vnlitvinov managed to merge [conda-forge/ray-packages-feedstock#8](https://github.com/conda-forge/ray-packages-feedstock/pull/8). Do you still need 1.3 specifically, or is going to 1.4 also an option?

Modin needs Ray 1.4 (1.2 and 1.3 have bugs preventing us from moving to them), but I still plan to make 1.3 a thing in conda-forge.

**Comment by devin-petersohn** [comment]
Upgrading to Ray 1.4 via #3132 .
